modular_skyrat/modules/modular_items/code/game/objects/items/miscellaneous.dm

```diff
@@ -7,8 +7,3 @@
 	throwforce = 3
 	w_class = WEIGHT_CLASS_NORMAL
 
-/obj/item/food/monkeycube/beno
```

Comment:
If you remove it try AT LEAST fix it.
Create a whole new mob for the monkey cube which you THINK is a problem.
For example, remove the ventcrawling flag or remove the ability to plant weeds
Maybe make weaker i dunno.

Reply:
its so easy fix things...